src/components/helpers.tsx

```diff
@@ -0,0 +1,16 @@
+import { css } from "styled-components"
+
+/**
+ * Helper function to display an element as a block that inherits its parents width
+ * @param margin value in pixels to remove from width 100%
+ */
+export const block = (margin: number = 0) => {
+    return props => {
+        if (props.block)
+            return css`
+                display: block;
+                width: calc(100% - ${margin}px);
+                margin: 10px auto;
+            `
+        }
```

Comment:
@damassi Yes for sure, I'll add those rules and make sure we enforce two spaces. I honestly assumed vscode's default setting was two spaces but in any case, I'll have that rule included in a subsequent PR.